### ISSUE TITLE
Move HUD location + Minimap Display Grey

### ISSUE THREAD

**Issue by SoreEyesRP** [joe-rogan-0001/joehud#17]
How can I move the HUD location from the bottom left side to the top middle of the screen?

Also, the minimap shows blank when I first get into a vehicle. I have to increase the size of the map and close it out for it to show in the minimap. This is obviously a bug? Is there a fix for this?

**Comment by Jayrcee** [comment]
did you ever get a responce on how to move the health bar @SoreEyesRP ?

**Comment by SoreEyesRP** [comment]
> did you ever get a responce on how to move the health bar @SoreEyesRP ?

No, I didn't. I ended up buying a HUD to suit our server needs.
